packages/common/core-utils/src/test/compare/compareJson.spec.ts

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { compareJson } from "../../compare";
+
+describe("deepCompareObjectEntries", () => {
```

Comment:
I'll admit these test cases are all over the place. I can clean up if needed but kind of inclined to just leave it as-is (unless there's some glaring omissions of important cases)